### ISSUE TITLE
The xml.etree.ElementTree module is deprecated.

### ISSUE THREAD

**Issue by ChakshuGupta13** [PetrVys/MotionPhoto2#11]
Since, `xml.etree.`ElementTree` is deprecated, hence, will it be better to switch to `defusedxml`?
> Deprecated since version 3.3: The xml.etree.ElementTree module is deprecated.

> Warning The [xml.etree.ElementTree](https://docs.python.org/3/library/xml.etree.elementtree.html#module-xml.etree.ElementTree) module is not secure against maliciously constructed data. If you need to parse untrusted or unauthenticated data see [XML vulnerabilities](https://docs.python.org/3/library/xml.html#xml-vulnerabilities).

> Using etree to parse untrusted XML data is known to be vulnerable to XML attacks. Replace etree with the equivalent defusedxml package. [https://bandit.readthedocs.io/en/latest/blacklists/blacklist_imports.html#b410-import-lxml](vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.esm.html)

**Comment by PetrVys** [comment]
Hello,

Thanks for the scan. I believe this is a false positive, since Tkd-Alex used lxml.etree (external library) and not xml.etree (internal Python C implementation). Lxml is a new implementation and to the best of my (limited) knowledge it is regularly updated and safe to use.

**Comment by ChakshuGupta13** [comment]
You're right about `xml.etree` - I confused it with `lxml`. But concern about `lxml` still stands. I have updated issue description with relevant link and reference; may that will help clarify.

**Comment by PetrVys** [comment]
Ok, I stand corrected :-)

I'll add it to the queue - in the meantime it's definitely not a serious issue, stealing data from malformed XMP tag into another offline picture is a very unlikely scenario ^_^
